app/models/user.rb

```diff
@@ -43,18 +28,6 @@ def friend_requests
     inverse_friendships.map { |fr| fr.user unless fr.status }.compact
   end
 
```

Comment:
>  def friend_requests
>     inverse_friendships.map { |fr| fr.user unless fr.status }.compact
>   end

isn't that the same as pending friends?, remember DRY

>   has_many :pending_friends, -> { where(friendships: { status: nil }) }, through: :friendships, source: :rqstuser

Reply:
After a code inspection, it's been detected that the scope pending_friends lacks the capability of taking the inverse_friendships attribute from user type objects, which is a necessary condition to retrieve the friend requests users list, because of this, it's been decided to remove this scope. 

@Usagib  @leonmezu1  :octocat: